sources/api/netdog/src/main.rs

```diff
@@ -49,11 +49,14 @@ static KERNEL_HOSTNAME: &str = "/proc/sys/kernel/hostname";
 static CURRENT_IP: &str = "/var/lib/netdog/current_ip";
 static KERNEL_CMDLINE: &str = "/proc/cmdline";
 static PRIMARY_INTERFACE: &str = "/var/lib/netdog/primary_interface";
+static PRIMARY_MAC_ADDRESS: &str = "/var/lib/netdog/primary_mac_address";
 static DEFAULT_NET_CONFIG_FILE: &str = "/var/lib/bottlerocket/net.toml";
 static OVERRIDE_NET_CONFIG_FILE: &str = "/var/lib/netdog/net.toml";
 static PRIMARY_SYSCTL_CONF: &str = "/etc/sysctl.d/90-primary_interface.conf";
+static SYSCTL_MARKER_FILE: &str = "/var/lib/netdog/primary_sysctls_set";
```

Comment:
This is a persistent path, which means the sysctls will be skipped after a reboot since the marker file will already exist.

Reply:
Previously the sysctl's were set with a system service that only ran once every boot.  The sysctl code is now in the "install" helper that wicked runs when setting up an interface as well as every time a DHCP lease is renewed.  Since this could happen often (especially for dhcp6), it doesn't really make sense to be setting and resetting the sysctls that often.

Reply:
I've fixed this by writing the marker file to `/run/netdog`